### ISSUE TITLE
False positives while linting and fixing sorted_imports

### ISSUE THREAD

**Issue by ZevEisenberg** [realm/SwiftLint#1380]
```swift
#if DEBUG
import Instabug // ← sorted_imports warns on this line
#endif
import OHHTTPStubs
import RZVinyl
import Swiftilities
```

SwiftLint 0.16.0

**Comment by ZevEisenberg** [comment]
Hmm… it seems to have resolved itself. I'm now wondering whether it's reporting on the wrong files, or something like that? It's reported several false positives in my app, and as I go through and clean up the true positives, the false ones seem to shift around.

**Comment by ZevEisenberg** [comment]
More followup: as long as there are true positives in the results, it seems like I'm also getting false positives. Or something like that. Unfortunately, I can't post the project I'm currently working in.

**Comment by ZevEisenberg** [comment]
To complicate things, Xcode wasn't showing all of the warnings that SwiftLint was outputting, so I had to run it in the command line and then look at each result in turn to find the true positives and fix them.

**Comment by ZevEisenberg** [comment]
[Example Project.zip](https://github.com/realm/SwiftLint/files/862661/Sorter.zip)

Here's an example of a project which demonstrates the missing positives, but not the false positives. Xcode shows only one warning, even though there are four files with unsorted imports. Could be an Xcode bug?

![screen shot 2017-03-22 at 3 41 13 pm](https://cloud.githubusercontent.com/assets/464574/24217370/fdf88184-0f15-11e7-88a3-a0bab85d4363.png)

**Comment by ZevEisenberg** [comment]
(I'll add that the `swiftlint` command line tool correctly shows all four warnings.)

**Comment by ZevEisenberg** [comment]
Erm, the false positives look like they're fixed on the latest code on the `master` branch. Feel free to close if this is known to be fixed, and I guess I'll just have to wait until the next release.
